tensorflow_datasets/image/chexpert.py

```diff
@@ -0,0 +1,114 @@
+"""https://stanfordmlgroup.github.io/competitions/chexpert/"""
+
+import csv
+import os
+
+import tensorflow_datasets.public_api as tfds
+import tensorflow as tf
+
+
+_DESCRIPTION = '''\
+CheXpert is a large dataset of chest X-rays and competition for automated chest 
+x-ray interpretation, which features uncertainty labels and radiologist-labeled 
+reference standard evaluation sets. It consists of 224,316 chest radiographs 
+of 65,240 patients, where the chest radiographic examinations and the associated 
+radiology reports were retrospectively collected from Stanford Hospital. Each 
+report was labeled for the presence of 14 observations as positive, negative, 
+or uncertain. We decided on the 14 observations based on the prevalence in the 
+reports and clinical relevance.
```

Comment:
This is where we'll want to make very clear that the user needs a special license.

Reply:
Sounds good 👍